js/src/agile-toolkit-ui.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable */
 /* global _ATKVERSION_:true, __webpack_public_path__:true */
-__webpack_public_path__ = window.bundlePublicPath === undefined ? '/public/' :  window.bundlePublicPath + '/';
+__webpack_public_path__ = window.__atkBundlePublicPath === undefined ? '/public/' :  window.__atkBundlePublicPath + '/';
```

Comment:
is there any reason why this must be under `windows` instead under `atk`?

Reply:
yes, because atk is not loaded yet and webpack will produce a proper import statement base on this value inside atk.

Reply:
hmm, I find it messy, but if browsers or some analysers do not complain about it, ok to keep...

Reply:
I know it looks messy but this needs to be set, in the main bundle entry file and prior to any import statement in order to work properly. And it does work well actually. It is needed for Saasty actually for it to properly load atk bundle file.

Reply:
Is everything ok?

Reply:
LGTM

Reply:
an approval would be nice then 😄